lib/Crypto/storedkeyrequest.h

```diff
@@ -26,15 +26,28 @@ class SAILFISH_CRYPTO_API StoredKeyRequest : public Sailfish::Crypto::Request
 {
     Q_OBJECT
     Q_PROPERTY(Sailfish::Crypto::Key::Identifier identifier READ identifier NOTIFY identifierChanged)
+    Q_PROPERTY(KeyComponents keyComponents READ keyComponents WRITE setKeyComponents NOTIFY keyComponentsChanged)
     Q_PROPERTY(Sailfish::Crypto::Key storedKey READ storedKey NOTIFY storedKeyChanged)
 
 public:
+    enum KeyComponent {
+        NoData          = 0,
+        MetaData        = 1,
+        PublicKeyData   = 2,
+        SecretKeyData   = 4
```

Comment:
The scenario when we combine `PublicKeyData` and `SecretKeyData` is possible, am I right?

Reply:
Of course, they're flags.  This is explicitly unit tested also.

Reply:
Okay, thanks.